syntax = "proto3";

package hello;

service Greeter {
    rpc SayHello (HelloRequest) returns (HelloReply) {}
    rpc Test (TestRequest) returns (TestReply) {}
}

message HelloRequest {
    string name1 = 1;
    int64 id = 2;
}

message HelloReply {
    string message = 1;
}

message TestRequest {
    string name1 = 1;
    int64 name2 = 2;
}

message TestReply {
    string message1 = 1;
}